### PR TITLE
don't actually enable ipv6 for local testing :-\

### PR DESCRIPTION
**Pull request by krzyzacy**
fixes unit test locally

/shrug
/assign @cjwagner @BenTheElder @leblancd

### CODE REVIEW

**Comment by leblancd** [comment]
@krzyzacy - Thanks for taking care of this! Apologies, I should have spotted this in my change. Your fix looks good to me. (Editorial nit: "enable" is misspelled in the PR title).

File: kubetest/kubeadmdind/kubeadm_dind.go [hunk 1]
```diff
@@ -113,7 +114,7 @@ func NewDeployer(control *process.Control) (*Deployer, error) {
 		// Valid value
 	case "ipv6", "dual-stack":
 		log.Printf("Enabling IPv6")
-		if err := d.run("sysctl -w net.ipv6.conf.all.disable_ipv6=0"); err != nil {
```
Comment (BenTheElder) [inline kubetest/kubeadmdind/kubeadm_dind.go]: :/ how did this get through?

Comment (krzyzacy) [reply kubetest/kubeadmdind/kubeadm_dind.go]: that somehow works in the kubekins container :-)

Comment (BenTheElder) [reply kubetest/kubeadmdind/kubeadm_dind.go]: we run dind containers with elevated privileges

Comment (krzyzacy) [reply kubetest/kubeadmdind/kubeadm_dind.go]: does bazel still require privilege containers?

Comment (BenTheElder) [reply kubetest/kubeadmdind/kubeadm_dind.go]: no, but dind does

Comment (krzyzacy) [reply kubetest/kubeadmdind/kubeadm_dind.go]: https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/test-infra/test-infra-presubmits.yaml#L23-L25

we set privilege for all our bazel jobs though :-/ maybe only for jobs that needs build?

Comment (BenTheElder) [reply kubetest/kubeadmdind/kubeadm_dind.go]: Bazel jobs used to, they shouldn't anymore.

Comment (BenTheElder) [reply kubetest/kubeadmdind/kubeadm_dind.go]: Somewhat orthogonal though, in general jobs have a different environment from local so this stuff has to be caught by review.

Comment (leblancd) [reply kubetest/kubeadmdind/kubeadm_dind.go]: Right. Enabling IPv6 in the kubekins container is fine (doesn't effect the host), but setting it locally when running UT is not good. I should have caught this when I moved the sysctl and added UT that exercised this. I can't think of any practical way to detect this in UT. Thanks for fixing this.

**Pull request merged**